src/styles/page-template/_template.njk

```diff
@@ -53,9 +51,13 @@
         <title>{{ page_title }}</title>
         <link rel="stylesheet" href="{{ assetsURL }}/css/{{ pageConfig.theme | default('main') }}.css">
         <link rel="stylesheet" media="print" href="{{ assetsURL }}/css/print.css">
-       
+        <meta name="theme-color" content="#3b7a9e"/>
```

Comment:
We probably want this to be a param in the config don't we? So it can be matched to the site's theme.